src/main/java/pb360/service/MessageObjectService.java

```diff
@@ -0,0 +1,9 @@
+package pb360.service;
+
+import pb360.model.*;
+
```

Comment:
should not import all

Reply:
i changed it to pb360.model.MessageObject only